codespell_lib/data/dictionary.txt

```diff
@@ -9696,9 +9743,12 @@ reonly->read-only
 reopended->reopened
 reoport->report
 reord->record
+reorer->reorder
 reorganision->reorganisation
 reosurces->resources
 reounded->rounded
+reources->resources
+reourcesn->resourcesn
```

Comment:
huh?

Reply:
Removed both.

Reply:
No need to remove the 1st one. The 2nd one was the issue.